### PR TITLE
Update Changelog and MongoDB docs

### PR DESCRIPTION
**Pull request by RassK**
### What

Follow up to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/421 
* Updates changelog
* Adds mongodb instrumentation info to instrumentations table
* Includes version column to instrumentations table

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.1.0-beta.1...HEAD)
 
+### Added
+
+- Adds Mongodb instrumentation support from .NET Core 3.1+.
```
Comment (Kielek) [inline CHANGELOG.md:12]:
```suggestion
- Adds MongoDB instrumentation support from .NET Core 3.1+.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.1.0-beta.1...HEAD)
 
+### Added
+
+- Adds MongoDB instrumentation support from .NET Core 3.1+.
```
Comment (pjanotti) [inline CHANGELOG.md:12]: If we use the suggestion below, can we get rid of the customization on the markdown lint?
```suggestion
- MongoDB instrumentation support from .NET Core 3.1+.
```

File: docs/config.md [hunk 3]
```diff
@@ -32,12 +32,14 @@ for more details.
...
 
-| ID | Library | Instrumentation type |
-|-|-|-|
-| `AspNet` | ASP.NET and ASP.NET Core | source |
-| `GraphQL` | [GraphQL](https://www.nuget.org/packages/GraphQL/) | bytecode |
-| `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | source |
-| `SqlClient` | [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient) and [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient) | source |
+| ID | Library | Supported versions | Instrumentation type |
+|-|-|-|-|
+| `AspNet` | ASP.NET MVC Framework | * | source |
```
Comment (pjanotti) [inline docs/config.md:37]: It seems that we need an issue to clarify those `*`. Is it something documented on their original repos?

**Review by pjanotti — COMMENTED** [review]
Could you please confirm if can get rid of the markdown lint change with the suggested re-word?

**Review by rajkumar-rangaraj — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (pjanotti) [reply CHANGELOG.md:12]: It doesn't like the "Added" as a title? Does want a name instead? Anyway, let's move ahead and refine it later.

**Review by pjanotti — APPROVED** [review]

File: docs/config.md (hunk 3 above)
Comment (pjanotti) [reply docs/config.md:37]: #636

**Review by pjanotti — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (pjanotti) [reply CHANGELOG.md:12]: #638

**Pull request merged**